Graphs/AbstractGraph.js

```diff
@@ -547,19 +547,29 @@ export default class AbstractGraph extends React.Component {
         return legend.circleSize * circleToPixel;
     }
 
-    renderLegend(data, legend, getColor, label, isVertical) {
+    renderLegend(data, legend, getColor, label, isVertical, heatmapColor) {
```

Comment:
heatmapColor = {}
pls fix other such code as well, null heatmapColor should not result in exception

Reply:
@killanch 

Changes have been made. Please check once.